rust/worker/src/execution/orchestration/compact.rs

```diff
@@ -106,7 +106,7 @@ impl CompactOrchestrator {
         };
         let input = PullLogsInput::new(collection_id, 0, 100, None, Some(end_timestamp));
         let task = wrap(operator, input, self_address);
-        match self.dispatcher.send(task).await {
+        match self.dispatcher.send(task, None).await {
```

Comment:
would we want to propagate here?

Reply:
Ah, yes! I had just orchestrated one e2e flow of `query_vectors` and this code is not in that path hence haven't touched this yet. There are plenty of other places where it is `None` for now but once I get more acquaintance with these code path, I'll add orchestration